topics/basics/indexing_and_psi_stubs/file_based_indexes.md

```diff
@@ -83,4 +83,4 @@ Generally, the word index should be accessed indirectly by using helper methods
 [`FilenameIndex`](upsource:///platform/indexing-api/src/com/intellij/psi/search/FilenameIndex.java) provides a quick way to find all files matching a specific file name.
 
 ### File Type Index
-[`FileTypeIndex`](upsource:///platform/indexing-api/src/com/intellij/psi/search/FileTypeIndex.java) serves a similar goal: it allows to find all files of a particular [`FileType`](upsource:///platform/core-api/src/com/intellij/openapi/fileTypes/FileType.java) quickly.
\ No newline at end of file
+[`FileTypeIndex`](upsource:///platform/indexing-api/src/com/intellij/psi/search/FileTypeIndex.java) serves a similar goal: it allows to find all files of a particular [`FileType`](upsource:///platform/core-api/src/com/intellij/openapi/fileTypes/FileType.java) quickly.
```

Comment:
I can't spot any difference in this sentence. Why is it marked as being different?

Reply:
There is some whitespace difference here. If you check this settings, it disappears:
![image](https://user-images.githubusercontent.com/19785715/108679346-709fc980-74fd-11eb-967b-3999a7adfa66.png)

I don't know why this whitespace was added/removed. I just used Github page to edit the file.